src/jp2image.cpp

```diff
@@ -655,12 +655,18 @@ static void boxes_check(size_t b,size_t m)
         auto p = reinterpret_cast<char*>(boxBuf.pData_);
         bool          bWroteColor = false ;
 
-        while ( count < length || !bWroteColor ) {
+        while ( count < length && !bWroteColor ) {
             enforce(sizeof(Jp2BoxHeader) <= length - count, Exiv2::kerCorruptedMetadata);
             auto pSubBox = reinterpret_cast<Jp2BoxHeader*>(p + count);
 
             // copy data.  pointer could be into a memory mapped file which we will decode!
+<<<<<<< HEAD
```

Comment:
nice

Reply:
Haha, I see it was [already fixed on main](https://github.com/Exiv2/exiv2/commit/d73dd0874dda6eaffe6326f10c0a4c540243753b).